packages/graphql-playground-react/src/components/Playground/ExplorerTabs/SideTabs.tsx

```diff
@@ -269,6 +267,7 @@ const ConnectedGraphDocs = connect<StateFromProps, DispatchFromProps, Props>(
   { forwardRef: true },
 )(SideTabs)
 
+// @ts-ignore
```

Comment:
I couldn't find anything actually using `SideTabs.Tab` but I'm not sure how this package is used and something could be depending on it.